tests/integration_tests/data_steward/cdr_cleaner/cleaning_rules/unit_normalization_test.py

```diff
@@ -143,7 +145,7 @@ def test_unit_normalization(self):
         tables_and_counts = [{
             'fq_table_name':
                 f'{self.project_id}.{self.dataset_id}.{MEASUREMENT}',
-            'loaded_ids': [1, 2, 3, 4, 5, 6, 7, 8, 9, 10],
+            'loaded_ids': [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11],
```

Comment:
Shall we also add the following to `tables_and_counts`? I recently learned that `sandbox_ids` are not tested if 'fq_sandbox_table_name' is not explicitly specified in `tables_and_counts`.

```
            'fq_sandbox_table_name':
                f'{self.project_id}.{self.sandbox_id}.{self.rule_instance.sandbox_table_for(MEASUREMENT)}',
```

Reply:
nice catch!  I added that to the integration test.